pages/u/[id].js

```diff
@@ -144,6 +144,16 @@ export default function Home() {
       ) : (
         <Magic />
       )}
+      <div className="flex items-center justify-center">
+        <p className="text-sm  text-gray-700">
+          {person.firstName} {person.lastName}?
+        </p>
+        <Link href={person.signedUp ? "/login" : "/register"}>
```

Comment:
there are no routes for /login or /regsiter:
this should be an anchor tag going to the sdr  website with the respective views.